release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1973,6 +1973,10 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+* When upgrading to {product-title} 4.9, the Cluster Version Operator blocks the upgrade for approximately five minutes while failing precondition checks. The error text, which says `It may not be safe to apply this update`, might be misleading. This error occurs when one or multiple precondition checks fail. In some situations, these precondition checks might only fail for a short period of time, for example, during an etcd backup. In these situations, the Cluster Version Operator and corresponding Operators will, by design, automatically resolve the failing precondition checks and the CVO successfully starts the upgrade.
++
+Users should check the status and conditions of their Cluster Version Operator. If the `It may not be safe to apply this update` error is displayed by the Cluster Version Operator, these statuses and conditions will provide more information about the severity of the message. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1999777[BZ#1999777], link:https://bugzilla.redhat.com/show_bug.cgi?id=2061444[BZ#2061444], link:https://bugzilla.redhat.com/show_bug.cgi?id=2006611[BZ#2006611].
```

Comment:
The only question I have is on the formatting of the BZs listed. Should they be in bold? And should they be in the parentheses (comma separated) like the others?

Reply:
Ohh they do not need ot be in parentheses because of the lead up to ("For more information, see. . ."). They do need to be bolded, though. I swear I thought when we said "For more information. . ." with BZs we we didn't put them in bold, but it looks like we do!